<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiberalEssentialOption</class>
 <widget class="QMainWindow" name="LiberalEssentialOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;
background-color: rgb(255, 208, 142);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_main_liberal_option_title">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>71</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>필수교양</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>149</y>
      <width>291</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBox_0_3">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>앙트레프레너십시대의 회계</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_0_4">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>한국사</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_0_5">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ACT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_right_arrow">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_right_arrow{
background-color: transparent;
border-image: url(:right_arrow3.png);
background: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>350</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_0_0">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>COMMUNICATION IN ENGLISH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_0_1">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>글쓰기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_0_2">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>창의와 소통</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_main_liberal_ment">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>이미 수강하신 필수교양을 선택하세요. </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_borderline">
    <property name="geometry">
     <rect>
      <x>-19</x>
      <y>-30</y>
      <width>897</width>
      <height>919</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/borderline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/카카오톡 받은 파일/gom.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_borderline</zorder>
   <zorder>label_main_liberal_option_title</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>pushButton_right_arrow</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_main_liberal_ment</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="menu_logo.qrc"/>
 </resources>
 <connections/>
</ui>
